### PR TITLE
Weather api PR

### PR DESCRIPTION
**Pull request by kbjude**
This Pull request has the features meant to be completed before raising a pull request. Among the features include:
- An input field for the user input
- Async function
- Fetching of the data from the API
- Toggle between °C and °F which comes with color changes and others

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @kbjude 
### You have done well so far :clap: However, you have a few issues to fix:-
- [ ] We recommend using es6 arrow functions, see below
```
const toggle = () => {
  # code
}
```
- [ ] Add more scripts in the package.json, see below
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open"
  },
```
### Make changes and submit for another code review. Good luck.
- Happy coding!

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    name.innerHTML = nameValue;
+    temperature.innerHTML = `Temperature: ${tempValue} °C`;
+    description.innerHTML = `Description: ${descdata}`;
+    humidity.innerHTML = `Humidity: ${humid}`;
+    img.src = `https://fernando-bc.com/weather-icons/${json.weather[0].icon}.svg`;
+    return response;
+  } catch (err) {
+    return err;
+  }
+}
```
Comment (rOluochKe) [inline src/index.js]: ### Refactor this code

**Comment by kbjude** [comment]
- All feedback added

**Review by christianotieno — APPROVED** [review]
### Project Approved 🍾 

Hello @kbjude 👋 
Congratulations on the hard work with the changes. keep it up 👏 
All the required changes have been fulfilled in this project. I will go ahead and mark it as complete 👍 
I wish you all the best in the next tasks 🥇 

### Happy coding 💻 

#### Christian Otieno (TSE)

**Pull request merged**